specs/vm/instruction_set.md

```diff
@@ -1417,6 +1418,22 @@ This modifies the `balanceRoot` field of the appropriate output.
 `messageID` is added to the `OutputMessage` Merkle tree as part of block header.
 TODO: document output messages merkle tree construction and maintenance and link here
 
+### SCWQ: State clear sequential 32 byte slots
+
+|             |                                                                               |
+|-------------|-------------------------------------------------------------------------------|
+| Description | A sequential series of 32 bytes is cleared from the current contract's state. |
+| Operation   | ```STATE[MEM[$rA, 32 * rB]] = None;```                                        |
+| Syntax      | `scwq $rA, $rB`                                                               |
+| Encoding    | `0x00 rA rB - -`                                                              |
+| Notes       |                                                                               |
+
+Panic if:
+
+- `$rA + 32 * rB` overflows
+- `$rA + 32 * rB > VM_MAX_RAM`
```

Comment:
Since we are clearing data from the database (state) instead of VM memory, it seems like this check isn't applicable.

Reply:
Those checks exist for SWW and SWWQ. Should those also not exist? Or are you saying that they are getting their input values from VM Memory so the check is necessary there but our input value is already known?

Reply:
Yeah the difference is that SWW and SWWQ are copying bytes from db into memory, and we need to ensure the memory range withing the VM is valid. 

Clearing contiguous state is different, because we are specifying data ranges inside of the DB instead of VM memory. I edited a previous comment to help clarify the intent of how I think this would be used but it might've been resolved before you saw it:

> The reason for this being that when data in sway is stored to the db, it is packed into contiguous 32 byte storage slots with an ascendingly increasing storage key. So if you wanted to remove an entire struct from the db in one operation, you could just provide the starting address and the number of slots that were used to store it.

https://github.com/FuelLabs/fuel-specs/pull/415#discussion_r993932158

Reply:
Also realized, you meant SRW and SRWQ right? For SWW and SWWQ, it doesn't seems like they have any `32 * d` type ops yet.